ERROR - 2021-11-23 05:23:26 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-23 05:23:26 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-23 05:31:02 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-23 05:31:02 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-23 05:40:20 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-23 05:40:20 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-23 10:51:36 --> include(apconn.php): Failed to open stream: No such file or directory
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\assets\aplogin.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(apconn....', 'C:\\xampp\\htdocs...', 3)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\assets\aplogin.php(3): include()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Cloth.php(19): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Cloth->Authentication()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cloth))
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-11-23 10:52:43 --> Call to undefined function mysqli_connect()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(230): include()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Common.php(1271): CodeIgniter\View\View->render('table', Array, true)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Cloth.php(15): view('table')
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Cloth->Table()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cloth))
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2021-11-23 13:53:19 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-23 13:53:19 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-23 13:53:50 --> Too few arguments to function App\Controllers\Blog::Post(), 0 passed in C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php on line 928 and exactly 1 expected
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->Post()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
